code/modules/hydroponics/grown/mushrooms.dm

```diff
@@ -309,7 +309,7 @@
 		
 /obj/item/seeds/fungus
 	name = "pack of cave fungus mycelium"
-	desc = "This mycelium grows into cave fungi, an edible variety of mushroom with anti-toxic properties."
+	desc = "This mycelium grows into cave fungi, an edible variety of mushroom with anti-toxic properties. (the anti toxic properties dont appear until grown)"
```

Comment:
```suggestion
	desc = "This mycelium grows into cave fungi, an edible variety of mushroom with anti-toxic properties. The anti-toxic properties don't appear until fully grown."
```
Why not just include it as the description as a part of the actual description instead of doing a weird ( ) inside the description?